.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
I wasn't aware of that behaviour. Although npm never seems to clean it up for me :/